skimage/metrics/set_metrics.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+
+from ._set_metrics import hausdorff_distance_onesided
+
+
+def hausdorff_distance(a, b):
```

Comment:
I think we can probably do better with these parameter names. For images we use `image0, image1`, or `image_true, image_test` so here I think we should use `points0, points1` (my preference) or `points_true, points_test`, or even `coords0, coords1`. Any thoughts on this @clementkng @scikit-image/core ?

Reply:
... Actually, I just saw that these are images, not actual points! So the docstring must be updated (suggestion below) and I would call these `image0` and `image1`.

Reply:
Done